.github/workflows/test-application.yaml

```diff
@@ -0,0 +1,66 @@
+name: Test application
+
+on:
+    pull_request:
+    push:
+        branches:
+            - 'master'
+            - '[0-9]+.x'
+            - '[0-9]+.[0-9]+'
+            - '[0-9]+.[0-9]+.x'
+
+jobs:
+    test:
+        name: 'PHP ${{ matrix.php-version }}, Symfony ${{ matrix.symfony-version }}'
+        runs-on: ubuntu-20.04
+
+        env:
+            SYMFONY_DEPRECATIONS_HELPER: ${{ matrix.symfony-deprecation-helper }}
+
+        strategy:
+            fail-fast: false
+            matrix:
+                include:
+                    - php-version: '7.1'
+                      composer-flags: '--prefer-lowest'
+                      symfony-version: '^3.4'
+                      symfony-deprecation-helper: 'weak'
+                      phpunit-version: 7.5
+
+                    - php-version: '7.2'
+                    - php-version: '7.3'
+                    - php-version: '7.4'
+                      symfony-version: '^4.4'
+                    - php-version: '7.4'
+                      symfony-version: '5.0.*'
+                    - php-version: '8.0'
+
+        steps:
+            - name: Checkout project
+              uses: actions/checkout@v2
+
+            - name: Install and configure PHP
+              uses: shivammathur/setup-php@v2
+              with:
+                  php-version: ${{ matrix.php-version }}
+                  tools: 'composer:v2'
+
+            - name: Require Specific Symfony Version
+              if: ${{ matrix.symfony-version }}
+              run: composer require --no-update symfony/symfony:${{ matrix.symfony-version }}
+
+            - name: Install dependencies with Composer
+              uses: ramsey/composer-install@v1
+              with:
+                  dependency-versions: ${{ matrix.dependencies }}
+                  composer-options: --prefer-dist --no-suggest
+
+            - name: Execute test cases
+              run: make test
+
+            - name: Execute test cases
+              run: |
+                  if [[ $SYMFONY_PHPUNIT_VERSION == '' ]]; then unset SYMFONY_PHPUNIT_VERSION; fi;
+                  vendor/bin/simple-phpunit ${{ matrix.phpunit-flags }}
+              env:
+                  SYMFONY_PHPUNIT_VERSION: ${{ matrix.phpunit-version }}
```

Comment:
we end up with two steps "Execute test cases" and both seem to run phpunit. i suggest to only have the one that does `make test` and configure the SYMFONY_PHPUNIT_VERSION for that step. the idea of the makefile is that it is easy to have the same behaviour locally as on the ci...

Reply:
oh yeah you are correct. seems like I did not need to set the SYMFONY_PHPUNIT_VERSION. as the phpunit bridge seems to use the correct version of  phpunit.

Reply:
it might randomly break when the phpunit bridge decides to use a newer phpunit version. but we can reintroduce the phpunit-version then, if needed.